libs/template/config.go

```diff
@@ -102,6 +102,27 @@ func (c *config) assignDefaultValues(r *renderer) error {
 // Prompts user for values for properties that do not have a value set yet
 func (c *config) promptForValues(r *renderer) error {
 	for _, p := range c.schema.OrderedProperties() {
+
+		// Skip prompting if required.
+		if p.Schema.SkipPromptIf != nil {
+			skip := true
+
+			// Check if conditions for skipping are satisfied
+			for name, property := range p.Schema.SkipPromptIf.Properties {
+				if v, ok := c.values[name]; ok && v == property.Const {
+					continue
+				}
+				skip = false
+				break
+			}
+
+			// Skip prompting and assign default value if conditions are satisfied
+			if skip {
+				c.values[p.Name] = p.Schema.Default
+				continue
+			}
+		}
```

Comment:
Please add a test for the skipping logic. I don't see coverage for it now.

Reply:
done